packs/data/equipment.db/throwers-bandolier.json

```diff
@@ -0,0 +1,61 @@
+{
+    "_id": "VUxPhU966hQaIq2C",
+    "img": "systems/pf2e/icons/default-icons/equipment.svg",
+    "name": "Thrower's Bandolier",
+    "system": {
+        "baseItem": null,
+        "containerId": null,
+        "description": {
+            "value": "<p>This bandolier is covered in straps and pouches capable of holding up to 2 Bulk of one-handed thrown weapons. A thrower's bandolier has a +1 weapon potency rune etched into it, and it can be etched with runes as though it were a one-handed thrown weapon. When you invest the thrower's bandolier, you can attune it to all the weapons sheathed in it (this ends any previous attunements made with the bandolier). Whenever you draw a weapon from the bandolier, the bandolier's runes are replicated onto that weapon. Any runes already on the weapon are suppressed, and any runes previously replicated to a different weapon in this way are removed, returning it to normal.</p>\n<p><strong>Activate</strong> <span class=\"pf2-icon\">1</span> envision, Interact</p>\n<hr />\n<p><strong>Effect</strong> All weapons attuned to the bandolier, not including any weapons you're currently wielding, return to the bandolier.</p>"
```

Comment:
```suggestion
            "value": "<p>This bandolier is covered in straps and pouches capable of holding up to 2 Bulk of one-handed thrown weapons. A thrower's bandolier has a +1 weapon potency rune etched into it, and it can be etched with runes as though it were a one-handed thrown weapon. When you invest the thrower's bandolier, you can attune it to all the weapons sheathed in it (this ends any previous attunements made with the bandolier). Whenever you draw a weapon from the bandolier, the bandolier's runes are replicated onto that weapon. Any runes already on the weapon are suppressed, and any runes previously replicated to a different weapon in this way are removed, returning it to normal.</p>\n<p><strong>Activate</strong> <span class=\"pf2-icon\">2</span> envision, Interact</p>\n<hr />\n<p><strong>Effect</strong> All weapons attuned to the bandolier, not including any weapons you're currently wielding, return to the bandolier.</p>"
```